backend/migrations/versions/999999999999_manually_add_delegation_for_one_of_users.py

```diff
@@ -0,0 +1,41 @@
+"""Manually add delegation for one of the users
+
+Revision ID: 999999999999
+Revises: 19f6b4f5ce2e
+Create Date: 2024-07-17 16:48:12.133033
+
+"""
+from alembic import op
+# from app.modules.common.delegation import get_hashed_addresses
+
+revision = "999999999999"
+down_revision = "19f6b4f5ce2e"
+branch_labels = None
+depends_on = None
+
+# primary = "0x2222222222222222222222222222222222222222" # redacted
```

Comment:
I think you can remove these commended out things

Reply:
+1